packages/legend-application-repl/src/components/dataCube/grid/menu/DataCubeGridMenu.tsx

```diff
@@ -89,33 +87,31 @@ export function buildGridMenu(
     },
     {
       name: 'Email',
-      menuItem: WIP_GridMenuItem,
-      cssClasses: ['!opacity-100'],
       subMenu: [
         {
           name: 'HTML',
-          menuItem: WIP_GridMenuItem,
-          cssClasses: ['!opacity-100'],
-          disabled: true,
+          action: () => {
+            dataCube.grid.generateEmail(true, true);
+          },
         },
         {
           name: 'Plain Text',
-          menuItem: WIP_GridMenuItem,
-          cssClasses: ['!opacity-100'],
-          disabled: true,
+          action: () => {
+            dataCube.grid.generateEmail(false, true);
```

Comment:
I believe these calls result in dangling unhandled promises, you can see the linter screaming, make sure to use the catch-all error handler that we have

```suggestion
            dataCube.grid.generateEmail(false, true).catch(application.logUnhandledError);
```